### PR TITLE
Feature/817 docs versioning

### PR DESCRIPTION
**Pull request by mitzimorris**
#### Submisison Checklist

- [ ] Run tests: `./runCmdStanTests.py src/test`
- [x] Declare copyright holder and open-source license: see below

#### Summary:
Simplified version number in latex docs and makefile - don't include patch, just <MAJOR>.<minor> version numbers.  The documentation should be valid throughout the release; it describes features and functionality in the release; patches are for bugfixes and hotfixes.

#### Intended Effect:
single CmdStan users guide valid for entirety of release.

#### Documentation:
udpated.

#### Copyright and Licensing

Please list the copyright holder for the work you are submitting (this will be you or your assignee, such as a university or company): Columbia University

By submitting this pull request, the copyright holder is agreeing to license the submitted work under the following licenses:
- Code: BSD 3-clause (https://opensource.org/licenses/BSD-3-Clause)
- Documentation: CC-BY 4.0 (https://creativecommons.org/licenses/by/4.0/)

### CODE REVIEW

**Comment by mitzimorris** [comment]
@serban-nicusor-toptal  - do the release scripts need to be changed as well?

File: makefile [hunk 1]
```diff
@@ -45,6 +45,7 @@ ifneq ($(filter-out clean clean-% print-% help help-% manual stan-update/% stan-
 endif
 
 CMDSTAN_VERSION := 2.22.0
+CMDSTAN_VERSION_DOC := 2.22
```
Comment (rok-cesnovar) [inline makefile]: Not related to this PR, but can I request a fix for the Cmdstan version on develop, the patch version was only applied to master. Line 47 here and https://github.com/stan-dev/cmdstan/blob/develop/.github/ISSUE_TEMPLATE.md are those that require the change. Thanks!

Comment (rok-cesnovar) [reply makefile]: I will let Nic review the rest, this is just a request. I can open a separate PR if you would rather do that, no problem.

**Comment by mitzimorris** [comment]
@rok-cesnovar - made changes - is this ready to go?

**Pull request merged**